src/libs/actions/Policy.js

```diff
@@ -498,6 +602,17 @@ function updateLastAccessedWorkspace(policyID) {
     Onyx.set(ONYXKEYS.LAST_ACCESSED_WORKSPACE_POLICY_ID, policyID);
 }
 
+/**
+ * loadFullPolicy of lastAccessedWorkspacePolicyID
+ */
+const fetchLastAccessedWorkspace = _.throttle(
+    () => lastAccessedWorkspacePolicyID && loadFullPolicy(lastAccessedWorkspacePolicyID), 10000, {leading: false, trailing: true},
+);
+
+// User could be viewing workspace setting page or member page on network reconnects
+// We call fetchLastAccessedWorkspace to update workspace data
+NetworkConnection.onReconnect(fetchLastAccessedWorkspace);
+
```

Comment:
We can decide this separately but I don't think these are related to the PR. Please revert.

Reply:
It is needed. Pending invitations will stuck if there is no Network Reconnect callback, when user is on the workspace or member page.
Or should I report another issue for this?

Reply:
I understood why this is used. But I don't think this is optimal. That's why it is important to discuss these changes before implementation.

Reply:
In the issue, you and I clearly stated to request policy on network reconnect. 
It is discussed there.

Reply:
All right, I will just remove it. And change QA steps. This will resolve @NikkiWines issue, because it will not support that action.